client-v2/src/components/CAPI/SearchQuery.tsx

```diff
@@ -30,15 +31,15 @@ class SearchQuery extends React.Component<
   };
 
   public static getDerivedStateFromProps(
-    { baseURL, fetch }: CAPISearchQueryProps,
+    { baseURL, fetch, isPreview }: CAPISearchQueryProps,
     prevState: CAPISearchQueryState
   ) {
     if (
       (baseURL && prevState.baseURL !== baseURL) ||
       (fetch && prevState.fetch !== fetch)
     ) {
       return {
-        capi: capiQuery(baseURL, fetch).search,
+        capi: isPreview ? capiQuery(baseURL, fetch).scheduled : capiQuery(baseURL, fetch).search,
```

Comment:
Since it makes to always query scheduled content when searching for preview I'm not too worried about having this logic in the search query component. It's annoying though that this also requires changing search params in another component.